RevenueCatUI/Templates/Components/TemplateComponentsView.swift

```diff
@@ -35,29 +35,28 @@ struct TemplateComponentsView: View {
         self.paywallComponentsData = paywallComponentsData
         self.onDismiss = onDismiss
 
+        // Step 0: Decide which ComponentsConfig to use (base is default)
+        let componentsConfig = paywallComponentsData.componentsConfigs.base
```

Comment:
Right now we are just always use that new base config thingy but... this would change depending on device type or orientation later on (however we decide its going to work 🤷‍♂️)